ckan/requirements.txt

```diff
@@ -22,7 +22,7 @@ ckanext-envvars==0.0.1
 -e git+https://github.com/ckan/ckanext-harvest.git@d00553831e20ae7c6737206ddb8a613b494f2b86#egg=ckanext-harvest
 -e git+https://github.com/ckan/ckanext-qa.git@d7d384cb18a85243ea62ef0bcc76bc1775f1c806#egg=ckanext-qa
 -e git+https://github.com/davidread/ckanext-report.git@b67875b2a5b4c9b9ab2cf255f074226255ca9398#egg=ckanext-report
--e git+https://github.com/keitaroinc/ckanext-saml2auth.git@d9d7db603fe41b2c4c35ab21fdaf8ea005ad5f7c#egg=ckanext-saml2auth
+-e git+https://github.com/keitaroinc/ckanext-saml2auth.git@a9bc575be31d2099650e6ca9a3b484bc0034717e#egg=ckanext-saml2auth
```

Comment:
```suggestion
-e git+https://github.com/keitaroinc/ckanext-saml2auth.git@ckan-2.8#egg=ckanext-saml2auth
```

Reply:
@FuhuXia the requirements.txt file is self-generated. We don't suppose to change it. We just change the `toml` file